src/Tizen.NUI.Components/Controls/Picker.cs

```diff
@@ -520,7 +520,10 @@ private void UpdateCurrentValue()
 
             if (loopEnabled) startY = ((dummyItemsForLoop + startItemIdx) * itemHeight) + startScrollOffset;
             // + 2 for non loop picker center align
-            else startY = ((2 + startItemIdx) * itemHeight) + startScrollOffset;
+            else {
```

Comment:
just small thing.
could you follow the indentation rule?
it is better to 
`else` 
`{`
thank you.

Reply:
Thank you. i updated PR.